packages/manager/src/utilities/sort-by.test.ts

```diff
@@ -0,0 +1,33 @@
+import { sortByVersion } from './sort-by';
+
+describe('sortByVersion', () => {
+  it('should identify the later major version as greater', () => {
+    const result = sortByVersion('2.0.0', '1.0.0', 'asc');
+    expect(result).toBeGreaterThan(0);
+  });
+
+  it('should identify the later minor version as greater', () => {
+    const result = sortByVersion('1.2.0', '1.1.0', 'asc');
+    expect(result).toBeGreaterThan(0);
+  });
+
+  it('should identify the later patch version as greater', () => {
+    const result = sortByVersion('1.1.2', '1.1.1', 'asc');
+    expect(result).toBeGreaterThan(0);
+  });
```

Comment:
Thanks again for the tests, these are great!

```suggestion
  });
  
  it('should identify the later minor version with differing number of digits', () => {
    const result = sortByVersion('1.30', '1.3', 'asc');
    expect(result).toBeGreaterThan(0);
  });
```

Maybe one extra test just to cover the "1.3" vs "1.30" scenario that led to this ticket